.github/CODEOWNERS

```diff
@@ -0,0 +1,179 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+# Owners can be @users, @org/teams or emails.
+
+# About
+
+# TODO
+
+# ----------------
+
+# Community
+
+# TODO
+
+## Asset Library
+
+# TODO
+
+# ----------------
+
+# Engine Details
+
+# TODO
+
+# ----------------
+
+# Getting Started
+
+# TODO
+
+# ----------------
+
+# Tutorials
+
+## 2D
+
+/tutorials/2d/                                      @godotengine/2d-nodes @godotengine/documentation
+
+# TODO: Any additional ownership for particles and rendering?
+
+## 3D
+
+/tutorials/3d/                                      @godotengine/3d-nodes @godotengine/documentation
+
+# TODO: Should any of these also have rendering ownership when rendering related?
+
+## Animation
+
+/tutorials/animation/                               @godotengine/animation @godotengine/documentation
+
+## Assets Pipeline
+
+/tutorials/assets_pipeline/                         @godotengine/import @godotengine/documentation
+
+# TODO: Any additional ownership for 3D and translations?
+
+## Audio
+
+/tutorials/audio/                                   @godotengine/audio @godotengine/documentation
+
+## Best Practices
+
+# TODO
+
+## Editor
+
+/tutorials/editor/*script*                          @godotengine/script-editor @godotengine/documentation
+
+# TODO
+
+## Export
+
+/tutorials/export/*android*                         @godotengine/android @godotengine/documentation
+/tutorials/export/*ios*                             @godotengine/ios @godotengine/documentation
+/tutorials/export/*linux*                           @godotengine/linux-bsd @godotengine/documentation
+/tutorials/export/*macos*                           @godotengine/macos @godotengine/documentation
+/tutorials/export/*visionos*                        @godotengine/ios @godotengine/documentation
```

Comment:
There isn't a specific team for this, so unsure what to add here, ownership is also missing from the main repo for this platform